model/version.go

```diff
@@ -412,7 +412,7 @@ func getMostRecentMainlineCommit(projectId string) (*Version, error) {
 func GetPreviousPageCommitOrderNumber(projectId string, order int, limit int, requesters []string) (*int, error) {
 	invalidRequesters, _ := utility.StringSliceSymmetricDifference(requesters, evergreen.SystemVersionRequesterTypes)
 	if len(invalidRequesters) > 0 {
-		return nil, errors.Errorf("invalid requesters: %v", invalidRequesters)
+		return nil, errors.Errorf("invalid requesters %s", invalidRequesters)
```

Comment:
`'%s'`

Reply:
This is a string slice so it will format as`[<elements>]` when used with `%s`.